test/MezzioInstallerTest/HomePageResponseTest.php

```diff
@@ -307,7 +318,7 @@ public function rendererlessInstallCasesProvider(): Generator
                 $routerName   = $this->expectedRouterAttributes[$routerClass]['routerName'];
                 $routerDocs   = $this->expectedRouterAttributes[$routerClass]['routerDocs'];
 
-                foreach ($this->intallTypes as $intallType) {
+                foreach ($this->installTypes as $intallType) {
```

Comment:
```suggestion
                foreach ($this->installTypes as $installType) {
```

The typo is here as well.